aws/resource_aws_transfer_server.go

```diff
@@ -30,6 +30,30 @@ func resourceAwsTransferServer() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validateArn,
+			},
+
+			"protocols": {
+				Type:     schema.TypeList,
+				MinItems: 1,
+				MaxItems: 3,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateFunc: validation.StringInSlice([]string{
```

Comment:
Can use AWS SDK-defined constants here (and elsewhere).

https://github.com/terraform-providers/terraform-provider-aws/blob/0f5fbb4d5262055d764087b800873ad02b6fb624/vendor/github.com/aws/aws-sdk-go/service/transfer/api.go#L5333-L5342